### PR TITLE
Update modules

### PR DESCRIPTION
**Pull request by akfaew**
Update to the latest version of ent.

### CODE REVIEW

**Comment by akfaew** [comment]
It seems that after upgrading the dependencies the build fails. I think this is a problem with github.com/vektah/gqlparser/v2 - can someone confirm?

File: go.mod [hunk 1]
```diff
@@ -3,39 +3,39 @@ module entgo.io/contrib
...
 	github.com/logrusorgru/aurora v2.0.3+incompatible // indirect
 	github.com/mattn/go-colorable v0.1.8 // indirect
-	github.com/mattn/go-sqlite3 v1.14.7
+	github.com/mattn/go-sqlite3 v1.14.8
 	github.com/mitchellh/mapstructure v1.4.1
 	github.com/oklog/ulid/v2 v2.0.2
+	github.com/pkg/errors v0.9.1 // indirect
 	github.com/stretchr/objx v0.3.0 // indirect
 	github.com/stretchr/testify v1.7.0
-	github.com/vektah/gqlparser/v2 v2.1.0
```
Comment (cliedeman) [inline go.mod:28]: keep this as is to fix the issue

**Comment by cliedeman** [comment]
Hi @akfaew It's a known issue with gqlgen unfortunately

**Comment by a8m** [comment]
Closing as modules have been upgraded since then.

**Pull request closed without merging**